<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EmployeeDialog</class>
 <widget class="QDialog" name="EmployeeDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>367</width>
    <height>456</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>EmployeeDialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="2" column="0">
    <widget class="QPushButton" name="regEmployeeButton">
     <property name="text">
      <string>Register</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QFrame" name="frame">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QFormLayout" name="formLayout">
      <item row="12" column="1">
       <widget class="QLineEdit" name="addressLineEdit">
        <property name="placeholderText">
         <string>Address</string>
        </property>
       </widget>
      </item>
      <item row="13" column="0">
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>Salary</string>
        </property>
       </widget>
      </item>
      <item row="18" column="0">
       <widget class="QLabel" name="label_7">
        <property name="text">
         <string>Gender</string>
        </property>
       </widget>
      </item>
      <item row="18" column="1">
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QCheckBox" name="maleCheckBox">
          <property name="text">
           <string>Male</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="femaleCheckBox">
          <property name="text">
           <string>Female</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="20" column="1">
       <widget class="QComboBox" name="comboBox">
        <item>
         <property name="text">
          <string>Janitor</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Doctor</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Nurse</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Receptionist </string>
         </property>
        </item>
       </widget>
      </item>
      <item row="13" column="1">
       <widget class="QLineEdit" name="salaryLineEdit">
        <property name="placeholderText">
         <string>Salary</string>
        </property>
       </widget>
      </item>
      <item row="12" column="0">
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>Address</string>
        </property>
       </widget>
      </item>
      <item row="11" column="1">
       <widget class="QLineEdit" name="ageLineEdit">
        <property name="placeholderText">
         <string>Age</string>
        </property>
       </widget>
      </item>
      <item row="11" column="0">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Age</string>
        </property>
       </widget>
      </item>
      <item row="10" column="1">
       <widget class="QLineEdit" name="lNameLineEdit">
        <property name="placeholderText">
         <string>Last Name</string>
        </property>
       </widget>
      </item>
      <item row="10" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Last Name</string>
        </property>
       </widget>
      </item>
      <item row="9" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>First Name</string>
        </property>
       </widget>
      </item>
      <item row="9" column="1">
       <widget class="QLineEdit" name="fNameLineEdit">
        <property name="placeholderText">
         <string>First Name</string>
        </property>
       </widget>
      </item>
      <item row="8" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>ID#</string>
        </property>
       </widget>
      </item>
      <item row="8" column="1">
       <widget class="QLineEdit" name="idLineEdit">
        <property name="placeholderText">
         <string>ID#</string>
        </property>
       </widget>
      </item>
      <item row="15" column="1">
       <widget class="QLineEdit" name="phoneNumLineEdit">
        <property name="placeholderText">
         <string>Phone Number</string>
        </property>
       </widget>
      </item>
      <item row="15" column="0">
       <widget class="QLabel" name="label_8">
        <property name="text">
         <string>Phone Number</string>
        </property>
       </widget>
      </item>
      <item row="20" column="0">
       <widget class="QLabel" name="label_10">
        <property name="text">
         <string>Position</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="label_9">
     <property name="text">
      <string>                        Employee Registration </string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
